src/D2L.Bmx/Aws/AwsClient.cs

```diff
@@ -1,14 +1,10 @@
-using System.Net;
-using System.Text;
-using System.Xml;
 using Amazon.SecurityToken;
 using Amazon.SecurityToken.Model;
 
 namespace D2L.Bmx.Aws;
 
 internal interface IAwsClient {
-	AwsRoleState GetRoles( string encodedSaml );
```

Comment:
Methods on an API client type should generally be about making API calls.
Parsing roles from the SAML response is purely internal processing, so moved it out.